src/tests_exhaustive.c

```diff
@@ -59,6 +59,19 @@ void random_fe(secp256k1_fe *x) {
         }
     } while(1);
 }
+
+static void random_fe_non_zero(secp256k1_fe *nz) {
+    int tries = 10;
+    while (--tries >= 0) {
+        random_fe(nz);
+        secp256k1_fe_normalize(nz);
+        if (!secp256k1_fe_is_zero(nz)) {
+            break;
+        }
+    }
+    /* Infinitesimal probability of spurious failure here */
+    CHECK(tries >= 0);
```

Comment:
(Is `random_fe` biased towards zero? I don't think so.)

Reply:
Okay, I was about to say: then just use `random_fe` and don't bother with zero at all? 

but if this is stolen from the other file, then let's not touch it, yes.